protocols/gossipsub/src/behaviour/tests.rs

```diff
@@ -38,9 +38,49 @@ mod tests {
     use crate::subscription_filter::WhitelistSubscriptionFilter;
     use crate::transform::{DataTransform, IdentityTransform};
     use crate::types::FastMessageId;
+    /* For debug purposes
+    use env_logger::{Builder, Env};
+      // Add this line to relevant tests.
+      Builder::from_env(Env::default().default_filter_or("info")).init();
```

Comment:
Why not just add `let _ = env_logger::try_init();` as the first line of each test, like we do in most other crates?

Reply:
Yeah, i'll modify this in the next iteration, if we decide to go down the `libp2p-metrics` route